.vscode/mcp.json

```diff
@@ -2,10 +2,11 @@
   "servers": {
     "expenses-mcp": {
       "type": "stdio",
-      "command": "${workspaceFolder}/src/.venv/bin/python",
+      "cwd": "${workspaceFolder}/src",
```

Comment:
Because VS Code expands workspaceFolder to the full path, it was hard to read mcp.json when it was in both places. Moving it to cwd means its only expanded on one line, and the other parameters are more readable.